themes-default/slim/src/components/display-show.vue

```diff
@@ -158,24 +174,28 @@ export default {
             });
         });
 
-        // Selects all visible episode checkboxes.
-        $(document.body).on('click', '.seriesCheck', () => {
-            $('.epCheck:visible').each((index, element) => {
-                element.checked = true;
-            });
-            $('.seasonCheck:visible').each((index, element) => {
-                element.checked = true;
-            });
+        // Selects all visible episode checkboxes
+        document.addEventListener('click', event => {
+            if (event.target && event.target.className.includes('clearAll')) {
+                [...document.getElementsByClassName('epCheck')].filter(isVisible).forEach(element => {
+                    element.checked = true;
+                });
+                [...document.getElementsByClassName('seasonCheck')].filter(isVisible).forEach(element => {
+                    element.checked = true;
+                });
```

Comment:
You *could* use this instead of the two loops, but it's probably going to be replaced later..
```js
[...document.querySelectorAll('.epCheck, .seasonCheck')].filter(isVisible).forEach(element => {
    element.checked = true;
});
```